taxi/tests/test-forms.py

```diff
@@ -0,0 +1,19 @@
+from django.test import TestCase
+
+from taxi.forms import DriverCreationForm
+
+
+class FormTests(TestCase):
+    def test_driver_creation_form_with_license_number_first_last_name_is_valid(
+            self):
+        form_data = {
+            "username": "new_user",
+            "password1": "user12test",
+            "password2": "user12test",
+            "first_name": "Test first",
+            "last_name": "Test last",
+            "license_number": "ABC12345",
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+        self.assertEqual(form.cleaned_data, form_data)

```

Comment:
The assertion `self.assertEqual(form.cleaned_data, form_data)` might fail because `cleaned_data` may not include all fields exactly as in `form_data` due to internal processing (e.g., password hashing). Consider checking only the fields that are expected to remain unchanged or use `form.cleaned_data.get('field_name')` for specific fields.